### ISSUE TITLE
How to fix OpenEphys .continuous with time discontinuities

### ISSUE THREAD

**Issue by weiglszonja** [open-ephys/open-ephys-python-tools#27]
Hey,

We have a similar [issue](https://github.com/catalystneuro/tye-lab-to-nwb/issues/40) to https://github.com/open-ephys/open-ephys-python-tools/issues/26, where some of the .continuous files have gaps during the recording. Since not all channels have gaps, I was wondering if the solution of @jsiegle still applies here:

> Hi Nick – You can use the same method I posted previously to trim both the beginning and end of the recording: https://github.com/open-ephys/open-ephys-python-tools/issues/25#issuecomment-1764962078

We need to ensure that the recording data is continuous (no gaps) after trimming, so would it work if
- iterate over all channels that have gaps
- find the longest continuous part that works for these channels
- trim all channels to this length
- save each channel into a new .continous file

**Comment by jsiegle** [comment]
Yes, that should work! Make sure to take the file header into account (see the note I just added to #26)
